macro/src/main/scala-3.x/monocle/macros/GenIso.scala

```diff
@@ -28,6 +29,7 @@ object GenIso {
   }
 
   /** Generate an [[Iso]] between an object `S` and `Unit`. */
+  @deprecated("no replacement", since = "3.1.0")
```

Comment:
You don't think this method is useful? I don't have a strong optinon.

Reply:
ah that's perfect, we can simply deprecate in favor of `Iso.fields`. If someone needs it, we can add an `Iso[ EmptyTyple, Unit]` somewhere.